### PR TITLE
Add instructions to run the project

### PR DESCRIPTION
**Pull request by KomalBhadani27**
Update ReadMe with instructions to run the project

### CODE REVIEW

**Comment by KomalBhadani27** [comment]
@telescopic You might want to add that **nltk** installation in the documentation. Feel free to suggest changes in the PR.

File: README.md [hunk 1]
```diff
@@ -5,11 +5,83 @@ Project to automatically convert news / entertainment websites into STAMP
 
 This is an internship project for Summer 2020.
 
-To get the Project working you will need to set the API key as an environment 
-variable with the name `GOOGLE_CLOUD_API_KEY` and the value as the base64 
-encoded string of the API key obtained from the console.
+**Project Requirements:**
```
Comment (detel) [inline README.md:8]:
```suggestion
##Project Requirements:##
```

File: README.md [hunk 2]
```diff
@@ -5,11 +5,83 @@ Project to automatically convert news / entertainment websites into STAMP
...
+
+To install all the dependencies used in the project, use command mentioned below:
+
+`pip install -r requirements.txt`
+
+**Note:** You need pip install installed in your system for running above command . To install pip refer the link attached below:
+
+https://pip.pypa.io/en/stable/installing/
+
+**How to run the project:**
```
Comment (detel) [inline README.md:34]:
```suggestion
##How to run the project:##
```

File: README.md [hunk 3]
```diff
@@ -5,11 +5,83 @@ Project to automatically convert news / entertainment websites into STAMP
...
+
+
+To install all the dependencies used in the project, use command mentioned below:
+
+`pip install -r requirements.txt`
+
+**Note:** You need pip install installed in your system for running above command . To install pip refer the link attached below:
+
+https://pip.pypa.io/en/stable/installing/
+
```
[the remark this answers is not in the document]
Comment (telescopic) [reply README.md:33]: @KomalBhadani27 @detel @tmadhuri @droidkid is this okay ? Do take a look at this!

File: README.md [hunk 4]
```diff
@@ -5,11 +5,92 @@ Project to automatically convert news / entertainment websites into STAMP
...
+        Refer to this link for more information on creating and setting API keys:
+        https://cloud.google.com/docs/authentication/api-keys
+    
+        To get the API key used for this project contact the owner of this repo.
+    
+    -  `FLASK_APP_SECRET_KEY` and the value could be any random generated string.
+
+        Refer to this link for more information on the usage of flask secret key:
+        https://flask.palletsprojects.com/en/1.1.x/quickstart/#sessions
```
Comment (detel) [inline README.md:23]: This should be under
### Set environment variables

File: README.md [hunk 5]
```diff
@@ -5,11 +5,92 @@ Project to automatically convert news / entertainment websites into STAMP
...
+
+    - Run this command in terminal:
+    
+        `$ python3 -m spacy download en_core_web_sm`
+  
+    - Run these commands in the python shell:
+
+        `>>> import nltk`
+    
+        `>>> nltk.download('punkt')`
```
Comment (detel) [inline README.md:44]: This should be under
```
### Install Dependencies
```

File: README.md [hunk 6]
```diff
@@ -5,11 +5,92 @@ Project to automatically convert news / entertainment websites into STAMP
...
+
+## How to run the project:
+
+This project can be run in two ways:
+
+- **Using Command Line Interface**
```
Comment (detel) [inline README.md:50]:
```suggestion
### Command Line Interface
```

File: README.md [hunk 7]
```diff
@@ -5,11 +5,92 @@ Project to automatically convert news / entertainment websites into STAMP
...
+  
+    `$ python3 main.py 'https://www.scoopwhoop.com/entertainment/memes-from-dd-ramayan/' 5`
+    
+    `$ python3 main.py 'https://www.scoopwhoop.com/entertainment/memes-from-dd-ramayan/'`
+  
+  **For more help, Use command:** `python3 main.py -h`
+  
+  **Note:** You must be in root directory of project (`stampify/`) to run above commands.
+
+- **Using Flask server**
```
Comment (detel) [inline README.md:76]:
```suggestion
###Flask Server
```

File: README.md (hunk 6 above)
Comment (detel) [reply README.md:50]: Remove Using from topic

File: README.md (hunk 4 above)
Comment (KomalBhadani27) [reply README.md:23]: Will be considered in coming iterations. Thanks for suggestion!

Comment (KomalBhadani27) [reply README.md:23]: Same for all other suggestions.

**Pull request merged**